t.js

```diff
@@ -0,0 +1,32 @@
+while (!!foo1) {
```

Comment:
Redundant double negation.

```suggestion
while (foo1) {
```